<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirSelectDialog</class>
 <widget class="QDialog" name="DirSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择一个目录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>名称</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>110</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>选择的目录路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>取消(&amp;C)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>选择(&amp;S)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>DirSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>DirSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
